astronomer/providers/amazon/aws/hooks/s3.py

```diff
@@ -45,6 +45,52 @@ async def _check_exact_key(client: AioBaseClient, bucket: str, key: str) -> bool
             else:
                 raise e
 
+    async def list_prefixes(
+        self,
+        client: AioBaseClient,
+        bucket_name: Optional[str] = None,
+        prefix: Optional[str] = None,
+        delimiter: Optional[str] = None,
+        page_size: Optional[int] = None,
+        max_items: Optional[int] = None,
+    ) -> List[Any]:
+        """
+        Lists prefixes in a bucket under prefix
+
+        :param client: ClientCreatorContext
+        :param bucket_name: the name of the bucket
+        :type bucket_name: str
+        :param prefix: a key prefix
+        :type prefix: str
+        :param delimiter: the delimiter marks key hierarchy.
+        :type delimiter: str
+        :param page_size: pagination size
+        :type page_size: int
+        :param max_items: maximum items to return
+        :type max_items: int
+        :return: a list of matched prefixes
+        :rtype: list
```

Comment:
```suggestion
        :param client: ClientCreatorContext
        :param bucket_name: the name of the bucket
        :param prefix: a key prefix
        :param delimiter: the delimiter marks key hierarchy.
        :param page_size: pagination size
        :param max_items: maximum items to return
        :return: a list of matched prefixes
```

We don't need type since we have Typehints :)